.github/workflows/python-package.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.9, 3.11]
```

Comment:
Is skipping some of these versions a good thing? In the past I have encountered problems where a specific version has a quirk. If we skip versions we migh miss this issue. I recommend having `3.7`-`3.8`-`3.9`-`3.10`. `3.11` is relatively new and we can maybe include it later when we drop `3.7`

Reply:
The reasoning behind skipping a version was this:
- If something does not work in 3.8, it would be because some feature was only available after 3.8. This would be caught by 3.7 too.
- Future versions of python will not break the existing code since they are meant to be backwards compatible (at least until python 4.0). The only cases where things can break is external libraries not supporting newer python versions.

But I agree, there may be some quirks with intermediate versions.

I'll make it `["3.7", "3.8", "3.9", "3.10", "3.11"]` since even [GitHub's starter workflow](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#using-the-python-starter-workflow) specifies all of them. 3.11 is nearly 5 months old and we should keep browser-history working with the latest python versions all the time.